annotation-utils/src/main/java/edu/isi/vista/annotationutils/ExtractAnnotationStats.kt

```diff
@@ -227,11 +405,69 @@ class ExtractAnnotationStats {
                             }
                         }
                     }
+                    h3 { +"Positive" }
+                    if (newAnnotationStats.byCorpusPositive != null) {
+                        table {
+                            tr {
+                                th { +"Project" }
+                                th { +"Total sentences (+)" }
+                                th { +"New sentences (+)" }
+                            }
+                            for (corpus in newAnnotationStats.byCorpusPositive.keys) {
+                                tr {
+                                    td { +"$corpus" }
+                                    td { +"${newAnnotationStats.byCorpusPositive[corpus]}" }
+                                    if (annotationDiffs == null) {
+                                        td { +"n/a" }
+                                    } else if (annotationDiffs.byCorpusPositive.isNullOrEmpty()) {
+                                        td { +"${newAnnotationStats.byCorpusPositive[corpus]}" }
+                                    } else {
+                                        td { +"${annotationDiffs.byCorpusPositive[corpus]}" }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                    h3 { +"Negative" }
+                    if (newAnnotationStats.byCorpusNegative != null) {
+                        table {
+                            tr {
+                                th { +"Project" }
+                                th { +"Total sentences (-)" }
+                                th { +"New sentences (-)" }
+                            }
+                            for (corpus in newAnnotationStats.byCorpusNegative.keys) {
+                                tr {
+                                    td { +"$corpus" }
+                                    td { +"${newAnnotationStats.byCorpusNegative[corpus]}" }
+                                    if (annotationDiffs == null) {
+                                        td { +"n/a" }
+                                    } else if (annotationDiffs.byCorpusNegative.isNullOrEmpty()) {
+                                        td { +"${newAnnotationStats.byCorpusNegative[corpus]}" }
+                                    } else {
+                                        td { +"${annotationDiffs.byCorpusNegative[corpus]}" }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                    table {
+                        tr {
+                            th { +"Project" }
+                            th { +"Total event types" }
```

Comment:
This is ambiguous to me, is it "total event types annotated all-time"?

Reply:
Yes; I will make these clearer